### PR TITLE
Update the HTML details element

### PR DESCRIPTION
**Pull request by mfuji09**
* Fixed the description of the open attribute. If the "false" value is set on this attribute, this element opens.
* Added a new style which uses the details[open] selector in an example - https://github.com/mdn/content/issues/239

### CODE REVIEW

File: files/en-us/web/html/element/details/index.html [hunk 1]
```diff
@@ -82,7 +82,8 @@ <h2 id="Attributes">Attributes</h2>
 
 <dl>
  <dt>{{htmlattrdef("open")}}</dt>
- <dd>This Boolean attribute indicates whether or not the details — that is, the contents of the <code>&lt;details&gt;</code> element — are currently visible. The default, <code>false</code>, means the details are not visible.</dd>
+ <dd>This Boolean attribute indicates whether or not the details — that is, the contents of the <code>&lt;details&gt;</code> element — are currently visible. By default this attribute is absent which means the details are not visible.<br>
```
Comment (ericwbailey) [inline files/en-us/web/html/element/details/index.html:85]: Just one question: What's the reasoning behind using a `br` in place of a paragraph? Seems like the note is a separate thought, so I'd think it'd be wrapped in `p` tags.

Comment (mfuji09) [reply files/en-us/web/html/element/details/index.html:85]: Good idea. It seems much beter. I try to do it.

**Comment by mfuji09** [comment]
@ericwbailey I have corrected it, so please review again.

**Review by ericwbailey — APPROVED** [review]
Nice, thank you!

**Pull request merged**